src/ImpurityDensities/BouleImpurityDensities/BouleImpurityDensities.jl

```diff
@@ -0,0 +1,10 @@
+"""
+Boule impurity densities are meant to be used when the impurity density is defined in the boule coordinates, where the z-axis is aligned with the boule growth direction. 
+
+Different models are provided. In each the field `det_z0` is the z-coordinate of the detector origin in boule coordinates. The z-direction of the detector is opposite to the z-direction of the boule coordinates.
+
+In this matter the detector impurities are automatically determined from those of the boule, depending on `det_z0`.
+"""
```

Comment:
This, to me, looks like something that should go into the documentation on impurity density models rather than here. This will not create a docstring as it is not preceding any function or struct.

Reply:
https://juliaphysics.github.io/SolidStateDetectors.jl/stable/man/electric_potential/#Impurity-Densities